packages/acceptance-tests/pkg-tests-specs/sources/commands/npm/info.test.ts

```diff
@@ -0,0 +1,47 @@
+import {tests} from 'pkg-tests-core';
+
+const npmInfo = async (run: tests.Run, ...args: Array<string>) => {
+  const {stdout} = await run(`npm`, `info`, `--json`, ...args);
+
+  return JSON.parse(stdout);
+};
+
+describe(`Commands`, () => {
+  describe(`npm info`, () => {
+    test(
+      `it should return information about the latest version of a package if no range is specified`,
+      makeTemporaryEnv({}, async ({path, run, source}) => {
+        await expect(npmInfo(run, `no-deps`)).resolves.toMatchObject({
```

Comment:
nit: I'd prefer we used a custom jest matcher (like [`toMatchJson`](https://github.com/duailibe/jest-json)) rather than a local helper which puts the command arguments "far" from the test.

Reply:
Done